11-checkout.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: lesson
+subtitle: Instructor Checkout Procedure
+---
+> ## Motivating Question
+What does the Instructor Checkout Procedure involve?
+{: .question}
+
+> ## Learning Objectives
+>
+> *   Learners will know the details of the Instructor Checkout Procedure.
+{: .objectives}
+
+## Discussion session
+You will be asked to join one of the discussion sessions (run as a video conference) focusing on the lesson of your choice from the Software and Data Carpentry curriculum. The session will be attended by a small group of instructor trainees like yourself and will be led by an experienced instructor. 
+
+This is an informal discussion for trainees to get more comfortable with the training material, clarify anything that may be unclear, learn (if you are still unsure) how to provide feedback and contribute to the materials.
+
+When working on the lesson, preparing for the discussion session, you can think about the following questions (which may also be used to structure the online discussion session):
+
+1. When working through the lesson did you feel like you would like to change the flow / present the concepts differently? If so, which ones and why?
+2. Was there anything that you found unclear in the material? Did you get stuck anywhere?
+3. What could be the misconceptions that the students could develop when learning this material? (i.e. which bits of the module may be tricky - if poorly explained, they may lead the students to broken mental model) 
+4. What assumptions about the prior knowledge would have for the students taking this module? If you were to list any prerequsites, what would they be? 
```

Comment:
I would reverse the order of these two sentences and would propose something like: "What prior knowledge do learners need to have before participating in this module" (by "module", do you mean lesson, or this some sub-unit?)